### ISSUE TITLE
error when run  sfgan_train_eval.py

### ISSUE THREAD

**Issue by Lithsun** [MLEnthusiast/SFGAN#1]
Dear Subhankar,

When I run sfgan_train_eval.py, there is one error and it will be good if you can give some suggestiosn to fix it.

File "sfgan_train_eval.py", line 288, in train
    x_inception_f=input_inception_fake)
 File "sfgan_train_eval.py", line 193, in model_loss
    g_model = generator(input_z, output_dim, alpha=alpha, size_mult=g_size_mult, reuse=True)
 File "sfgan_train_eval.py", line 114, in generator
    x2 = tf.layers.conv2d_transpose(x1, size_mult * 2, 5, strides=2, padding='same')

InvalidArgumentError (see above for traceback): Conv2DCustomBackpropInput: Size of out_backprop doesn't match computed: actual = 4, computed = 128 spatial_dim: 1 input: 256 filter: 5 output: 4 stride: 2 dilation: 1

 [[Node: generator_1/conv2d_transpose/conv2d_transpose_2 = _MklConv2DBackpropInput[T=DT_FLOAT, _kernel="MklOp", data_format="NHWC", padding="SAME", strides=[1, 2, 2, 1], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:CPU:0"](generator_1/conv2d_transpose/stack, generator/conv2d_transpose/kernel/read, generator_1/Maximum, DMT/_189, generator/conv2d_transpose/kernel/read:1, generator_1/Maximum:1)]]

**Comment by MLEnthusiast** [comment]
Dear @Lithsun ,

Are you running the code on your cpu? Because from your log it says _device="/job:localhost/replica:0/task:0/device:CPU:0".
